cluster/gce/gci/configure-helper.sh

```diff
@@ -2918,7 +2937,7 @@ function main() {
     start-kube-controller-manager
     start-kube-scheduler
     wait-till-apiserver-ready
-    start-kube-addons
+    start-kube-addons "$@"
```

Comment:
why do we need this?

Reply:
The way I understood it, the line where `start-kube-addons` does

`setup-node-termination-handler-manifest "$@"`

it is the script's inparameters that are used. But perhaps it is not, but rather eventual optional parameters sent to `start-kube-addons` and this line above should rather be

`start-kube-addons ''`

?